src/completion/CamelApplicationLauncherTasksCompletionItemProvider.ts

```diff
@@ -163,7 +163,7 @@ export class CamelApplicationLauncherTasksCompletionItemProvider implements vsco
 	"command": "jbang", // jbang binary must be available on command-line
 	"args": [
 		"'-Dorg.apache.camel.debugger.suspend=true'", // requires Camel 3.18+ to take effect
-		"'-Dcamel.jbang.version=4.5.0'", // to adapt to your Camel version. 3.16+ is required
+		"'-Dcamel.jbang.version=4.13.0'", // to adapt to your Camel version. 3.16+ is required
```

Comment:
4.14 can be used already

Reply:
currently the default version is 4.13, I prefer to change it when upgrading default to 4.14 https://github.com/camel-tooling/camel-dap-client-vscode/pull/927